prime-router/metadata/schemas/CA/ca-scc-covid-19.schema

```diff
@@ -0,0 +1,205 @@
+name: "ca-scc-covid-19"
+topic: "covid-19"
+description: "CSV download schema for Santa Clara County Public Health Department"
+trackingElement: filler_order_id
+basedOn: "covid-19"
+elements:
+  - name: "ordering_facility_name"
+    csvFields:
+      - name: "Facility Name"
+
+  - name: "testing_lab_clia"
+    csvFields:
+      - name: "Facility CLIA"
+
+  - name: "ordering_facility_street"
+    csvFields:
+      - name: "Facility Street Address"
+
+  - name: "ordering_facility_city"
+    csvFields:
+      - name: "Facility City"
+
+  - name: "ordering_facility_state"
+    csvFields:
+      - name: "Facility State"
+
+  - name: "ordering_facility_zip_code"
+    csvFields:
+      - name: "Facility Zip"
+
+  - name: "ordering_facility_phone_number"
+    csvFields:
+      - name: "Facility Phone"
+
+  - name: "patient_id"
+    csvFields:
+      - name: "Patient Identifier"
+
+  - name: "patient_first_name"
+    csvFields:
+      - name: "Patient First Name"
+
+  - name: "patient_middle_initial"
+    csvFields:
+      - name: "Patient Middle Initial"
+
+  - name: "patient_last_name"
+    csvFields:
+      - name: "Patient Last Name"
+
+  - name: "patient_dob"
+    csvFields:
+      - name: "Patient Date of Birth"
+
+  - name: "patient_gender"
+    csvFields:
+      - name: "Patient Sex"
+
+  - name: "patient_race"
+    csvFields:
+      - name: "Race"
+        format: "$code"
+
+  - name: "patient_ethnicity"
+    csvFields:
+      - name: "Ethnicity"
+        format: "$code"
+
+  - name: "local_Language"
+    default: ""
```

Comment:
for the purposes of faking data, all elements should have a type defined, or you will encounter the error on line 170 in `FakeReport` about invalid element type.